docs/quickstart.rst

```diff
@@ -53,28 +53,33 @@ A step by step example
    output statistics about the newly created archive such as the amount of unique
    data (not shared with other archives)::
 
+    ------------------------------------------------------------------------------
     Archive name: Tuesday
-    Archive fingerprint: 387a5e3f9b0e792e91ce87134b0f4bfe17677d9248cb5337f3fbf3a8e157942a
-    Start time: Tue Mar 25 12:00:10 2014
-    End time:   Tue Mar 25 12:00:10 2014
-    Duration: 0.08 seconds
-    Number of files: 358
-                           Original size      Compressed size    Deduplicated size
-    This archive:               57.16 MB             46.78 MB            151.67 kB
-    All archives:              114.02 MB             93.46 MB             44.81 MB
-
+    Archive fingerprint: bd31004d58f51ea06ff735d2e5ac49376901b21d58035f8fb05dbf866566e3c2
+    Start time: Tue Feb 16 18:15:11 2016
+    End time: Mon Feb 16 18:15:11 2016
```

Comment:
oops, looks like this will need another fix later as i did some post-rc1 changes.